docs/docs/nlu-only.mdx

```diff
@@ -51,3 +51,31 @@ To do this, run:
 ```bash
 curl localhost:5005/model/parse -d '{"text":"hello"}'
 ```
+
+## Connecting to an NLU server
+
+
+You can connect a Rasa dialogue management only server with a separately running NLU server
+by adding the connection details to the dialogue management server's endpoint configuration file:
+
+```yaml-rasa title="endpoints.yml"
+nlu:
+    url: "http://<your nlu host>:<your nlu port>"
+    token: <token>  # [optional]
+    token_name: <name of the token> # [optional] (default: token)
+```
+
+The `token` and `token_name` refer to optional [authentication parameters](./http-api.mdx#token-based-auth).
+
+The dialogue management server should serve a model that does not include an NLU model.
+To obtain a dialogue management only model, train a model with `rasa train core` or use 
+`rasa train` but exclude all NLU data.
+
+When the dialogue management server receives a message, it will [send a request](https://rasa.com/docs/rasa/pages/http-api#operation/parseModelMessage) to 
+`http://<your nlu host>:<your nlu port>/model/parse` and use the parsing information returned.
+
+Note that you should **not** use the same endpoint configuration file for the NLU server as for the dialogue management server, since
+the `nlu` endpoint refers back to the NLU server itself.
```

Comment:
This is a bit confusing to me as written -- but if you provide it does it get mad? I would think NLU-only doesn't even load up endpoints 🤔

Reply:
It does get mad, because it tries to circular-reference itself - which is very confusing. I'll add a better explanation